cni/network/network.go

```diff
@@ -1052,11 +1056,13 @@ func (plugin *NetPlugin) Delete(args *cniSkel.CmdArgs) error {
 				// if cni hits this, mostly state file would be missing and it can be reboot scenario where
 				// container runtime tries to delete and create pods which existed before reboot.
 				err = nil
-				return err
+				if !plugin.nm.IsStatelessCNIMode() {
```

Comment:
nit: Can we add a comment here why we need to do this for stateless CNI ?

Reply:
addressed in the next commit